modules/ml/src/main/java/org/apache/ignite/ml/tree/randomforest/data/NodeId.java

```diff
@@ -27,6 +28,12 @@
     /** Serial version uid. */
     private static final long serialVersionUID = 4400852013136423333L;
 
+    /**
+     * Empty constructor required by {@link Externalizable}.
+     */
+    public NodeId() {
```

Comment:
What about update this class so that it doesn't extend `BiTuple` and just make this class `Serializable`?

Reply:
ok